app/views/spree/admin/shared/_new_resource_links.html.haml

```diff
@@ -0,0 +1,4 @@
+.form-buttons.filter-actions.actions
+  = button t(:create), 'icon-ok'
```

Comment:
Looking at the code now, to fix the issue, all you needed would be to change the translation calls from Spree.t to t. because spree.actions.create and spree.actions.cancel are not there in the en.yml file but both are under actions.create (Spree.t is similar to t. but adds spree in the namespace).

Here you are moving to root level :create and :cancel. I think it's ok, I would prefer to keep these under the actions namespace, so I'd use t('actions.create') and t('actions.cancel') instead of t(:create) and t(:cancel).
The OR can stay as t(:or).

Reply:
Thank you very much - that makes complete sense. We went ahead and updated with the change.